new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -264,6 +299,46 @@ public static boolean appAccessibleInPodKubectl(
     return Application.appAccessibleInPodKubectl(namespace, podName, port, appPath, expectedResponse);
   }
 
+  /**
+   * Check if the given WebLogic credentials are valid by using the credentials to 
+   * invoke a RESTful Management Services command.
+   *
+   * @param host hostname of the admin server pod
+   * @param podName name of the admin server pod
+   * @param namespace name of the namespace that the pod is running in
+   * @param username WebLogic admin username
+   * @param password WebLogic admin password
+   * @return true if the RESTful Management Services command succeeded
+   */
+  public static Callable<Boolean> credentialsValid(
+      String host,
+      String podName,
+      String namespace,
+      String username,
+      String password) {
+    return () -> Application.credentialsValid(host, podName, namespace, username, password);
```

Comment:
Should it not go the Domain Impl,  the Application has nothing to do with Credential Validation.  

A general Comments:  The credential update/Validation is a one-off usecase, which will not be used in other MII usecaes.  So it may NOT be a candidate for a generic assertion at the utility level. We can write the assertion inside the TestClass to limit its scope.

Reply:
A general rule is that the test code should not use action/assertion impl classes.
I can move it to Domain.java.